nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -128,6 +128,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The [services.unifi-video.openFirewall](#opt-services.unifi-video.openFirewall) module option default value has been changed from `true` to `false`. You will need to explicitly set this option to `true`, or configure your firewall.
 
+- The option `i18n.inputMethod.fcitx5.enableRimeData` has been removed. Default RIME data is now included in `fcitx5-rime` by default, and can be customized using `fcitx5-rime.override { rimeData = [ package1, package2, ... ]; }`
```

Comment:
The option `i18n.inputMethod.fcitx5.enableRimeData` has been removed. Default RIME data `pkgs.rime-data` is now included in `fcitx5-rime` by default, and can be customized using `fcitx5-rime.override { rimeData = [ pkgs.rime-data, package2, ... ]; }`
Explicit default rime data as `pkgs.rime-data` to avoid confusion and to ease migration.

Reply:
Fixed as suggested.